src/rlx_prv_relup.erl

```diff
@@ -145,7 +155,14 @@ make_upfrom_script(State, Release, UpFrom) ->
                             "relup successfully created!"),
             {ok, State};
         {ok,_, Module,Warnings} ->
-            ?RLX_ERROR({relup_script_generation_warn, Module, Warnings});
+            case WarningsAsErrors of
```

Comment:
`warnings_as_errors` is not set: relup is generated, `systools:make_relup` returns `{ok,_, Module,Warnings}` warnings are emitted
`warnings_as_errors` is set: relup is not generated, `systools:make_relup` returns `error` atom, in this case i would expect to receive `{error,Module,Errors}`, since no relup is generated there is no danger of overwriting a previous one